---a/net/minecraft/world/biome/BiomeGenHell.java
+++b/net/minecraft/world/biome/BiomeGenHell.java
@@ -4,12 +4,12 @@
 import net.minecraft.entity.monster.EntityMagmaCube;
 import net.minecraft.entity.monster.EntityPigZombie;
 
-public class BiomeGenHell extends BiomeGenBase
-{
+public class BiomeGenHell extends BiomeGenBase {
+
     private static final String __OBFID = "CL_00000173";
 
-    public BiomeGenHell(int p_i1981_1_)
-    {
+    public BiomeGenHell(int p_i1981_1_) {
         super(p_i1981_1_);
         this.spawnableMonsterList.clear();
         this.spawnableCreatureList.clear();
@@ -19,4 +19,4 @@
         this.spawnableMonsterList.add(new BiomeGenBase.SpawnListEntry(EntityPigZombie.class, 100, 4, 4));
         this.spawnableMonsterList.add(new BiomeGenBase.SpawnListEntry(EntityMagmaCube.class, 1, 4, 4));
     }
-}
+}
